style/_colors.scss

```diff
@@ -90,6 +96,9 @@ $semantic-colors: (
     &.sd-outline-#{$color}:hover {
       border-color: var(--sd-color-#{$color}-highlight) !important;
     }
+    &.sd-outline-#{$color}:focus-visible {
+      outline-color: var(--sd-color-#{$color}-highlight) !important;
+    }
```

Comment:
This file sets the focus ring color to match either the border color or the background color of links and buttons.